src/edit-image.tsx

```diff
@@ -158,7 +158,7 @@ export default function EditImage(
                     onClick={() => inputEl?.current?.click()} />}
             </div>
             <div>מילות חיפוש</div>
-            <input type="text" value={keywords} onChange={(e) => { }} />
+            <input type="text" value={keywords} onChange={(e) => { setKeywords([e.target.value]) }} />
```

Comment:
if keywords is an array of strings, this seems wrong. as you just put as the first element all keywords?

Reply:
it is saved as array of string in media.tx line 15 I do the split
 const _save = useCallback((imageInfo: MediaResource, file?: File) => {
so when the user inserted the data it is array with  only 1 element "oneley,secondkey..." and when I do the save I do the split